int32[] sensor_value
